plugins/ocm/package.json

```diff
@@ -39,7 +39,8 @@
     "@material-ui/lab": "^4.0.0-alpha.45",
     "@patternfly/patternfly": "^5.1.0",
     "@patternfly/react-icons": "^5.1.1",
-    "react-use": "^17.4.0"
+    "react-use": "^17.4.0",
+    "semver": "^7.5.4"
```

Comment:
@Zaperex just wanted to confirm if the package addition here didn't have any yarn.lock updates?

Reply:
The `semver` package should already be included in the `yarn.lock` so regeneration shouldn't be required